src/scss/_projects.scss

```diff
@@ -1,6 +1,9 @@
 .projects {
-    &__wrapper {
-        background-color: $h1-font-color;
+    margin: 150px 0 115px 0;
+    /*надо переводить в % (wh) или em и делать одинаковым для каждой section*/
```

Comment:
great idea.
You can create a general class for this, it could be something like `section` or `sc`
and the styles for this class will be
`margin: 150px 0 115px 0`